src/components/filters/FilterBy/PriceFilter.jsx

```diff
@@ -9,15 +12,15 @@ export default function PriceFilter() {
             type="range"
             min="1000"
             max="5000"
-            // value={state.priceLimit}
+            value={stateFilter.priceLimit}
             list="tickmarks"
             step="1000"
```

Comment:
I saw that u have many products below 1000. So, u may want to start from a lower number.